gui/src/ScenarioSelector.tsx

```diff
@@ -256,7 +301,7 @@ const ScenarioSelector = (props: ScenarioSelectorProps) => {
     <div>
       <Box
         sx={{
-          maxWidth: 300,
+          maxWidth: 250,
```

Comment:
This I suggest could be a const declare high in the file.... easier to spot/modify

Reply:
this is part of the changes I brought